hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/OzoneAcl.java

```diff
@@ -97,9 +97,23 @@ public OzoneAcl(ACLIdentityType type, String name, ACLType acl,
    *
    * @param type   - Type
    * @param name   - Name of user
-   * @param acls   - Rights
    * @param scope  - AclScope
+   * @param acls   - Rights
    */
+  public OzoneAcl(ACLIdentityType type, String name, AclScope scope, ACLType... acls) {
+    this(type, name, bitSetOf(acls), scope);
+  }
+
+  private static BitSet bitSetOf(ACLType... acls) {
+    BitSet bits = new BitSet();
+    if (acls != null && acls.length > 0) {
```

Comment:
```suggestion
    if (acls != null) {
```

Reply:
What's the problem with `acls.length > 0`?

Reply:
The length > 0 is handled implicitly by the for loop, so its not really needed. I guess this is a nit - change it if there are other comments, otherwise it will do no harm.

Reply:
I know that it's not strictly necessary.  I have two problems:

1. Don't know the runtime cost of creating an iterator for the empty array in the for loop behind the scene.  Maybe the compiler optimizes it away.
2. Running full CI again for such a cosmetic change does not seem worth the effort.  Especially when the reviewer cannot spend the extra 10 seconds to write "nit:" or something like that.